### ISSUE TITLE
Strange loss cure for Tactron2 TTS?

### ISSUE THREAD

**Issue by fly12357** [espnet/espnet#1833]
I train the Tactron2 TTS model on my own dataset, but the loss curve  increase sharply at some epoch. 
![image](https://user-images.githubusercontent.com/40494145/79184274-42198280-7e46-11ea-95c3-eed0c1112320.png)

Have you ever meet this case?

**Comment by unilight** [comment]
In my experience, this happens when your training set is too small or the learning rate is too large. Try tuning some hyperparameters like learning rate, batch size, etc.

**Comment by fly12357** [comment]
> In my experience, this happens when your training set is too small or the learning rate is too large. Try tuning some hyperparameters like learning rate, batch size, etc.

Thanks for your reply. I'll try update my hyperparameters.

**Comment by tuanhuu162** [comment]
@fly12357 I am facing the same issue with the strange loss curve for Tacotron2 although I have already trimmed the audio. I am using 12 hours of audio data for training with each sample long 3 seconds. I trained on epoch 200 and the data is a lot of noise at first. Do you solve the problem yet?
